cli/assets.go

```diff
@@ -256,3 +279,325 @@ func deleteAssetByIDCommand() *cobra.Command {
 
 	return cmd
 }
+
+func listAllTypesCommand() *cobra.Command {
+	var types, services, data, q, q_fields string
+	cmd := &cobra.Command{
+		Use:   "types",
+		Short: "lists all asset types",
+		Example: heredoc.Doc(`
+			$ compass asset types
+			$ compass asset types -t type1 --query query1 --query_fields q_fields1
+		`),
+		Args: cobra.NoArgs,
+		Annotations: map[string]string{
+			"action:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			spinner := printer.Spin("")
+			defer spinner.Stop()
+
+			clnt, cancel, err := client.Create(cmd.Context())
```

Comment:
Docs for `cmd.Context()`.
```
// Context returns underlying command context. If command wasn't
// executed with ExecuteContext Context returns Background context.
```

We aren't executing the command with `ExecuteContext` currently, please fix.